docker-compose.yml

```diff
@@ -53,17 +53,25 @@ services:
     networks:
       - stacks-blockchain
 
+  redis:
+    # TODO protect via password
```

Comment:
we can do that TODO here or in a separate pr

Reply:
Let's leave this separate, already created issue : https://github.com/sigle/stxstats/issues/33